src/adapter/src/coord/message_handler.rs

```diff
@@ -254,6 +254,40 @@ impl<S: Append + 'static> Coordinator<S> {
                         .await;
                 }
             }
+            ControllerResponse::ComputeReplicaWriteFrontiers(updates) => {
```

Comment:
I think this is fine for now, but repeating this logic for everything we want to log is getting a bit annoying ... (mainly my fault as I wrote the initial one). You don't have to change anything now, but later we can try to think of a way to refactor it.

Reply:
Agree! We can probably implement a generic status update and merge these cases into one (also mentioned this in the out of scope section of the issue #15996 :nerd_face:  )